iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -679,12 +679,21 @@ private static Network initNetwork(NetworkFactory networkFactory, NetworkDeseria
         network.setCaseDate(date);
         network.setForecastDistance(forecastDistance);
 
-        ValidationLevel[] minValidationLevel = new ValidationLevel[1];
-        minValidationLevel[0] = ValidationLevel.STEADY_STATE_HYPOTHESIS;
-        IidmSerDeUtil.runFromMinimumVersion(IidmVersion.V_1_7, context, () -> minValidationLevel[0] = reader.readEnumAttribute(MINIMUM_VALIDATION_LEVEL, ValidationLevel.class));
+        ValidationLevel minValidationLevel;
+        if (context.getOptions().getMinimalValidationLevel().isPresent()) {
+            minValidationLevel = context.getOptions().getMinimalValidationLevel().orElseThrow();
```

Comment:
Specify the exception thrown in `orElseThrow`

Reply:
The else part couldn't be reached because of the previous check.
I introduced a variable (to avoid a Sonar issue) and used `.get()` instead of `.orElseThrow()`.